roles/docker-service/README.md

```diff
@@ -0,0 +1,122 @@
+docker-service
+--------------
+
+This module is an extension of the docker compose formula allowing the formula to deliver an ecs service or a docker compose file.
```

Comment:
This entire role needs to be updated to play nicely with ansilble, was committed so I don't lose it while I migrate all the other things